connector/protobuf/src/test/resources/protobuf/basicmessage_noimports.desc

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
Where is this used?
Did you mean to another test?

Reply:
@rangadi yes, it is used in the unit test above, how else can we raise the CANNOT_CONSTRUCT_PROTOBUF_DESCRIPTOR exception?

Reply:
Right. I didn't see the second test carefully.